examples/single_chip/camera_web_server/main/Kconfig.projbuild

```diff
@@ -43,6 +43,8 @@ config CAMERA_MODEL_WROVER_KIT
     bool "WROVER-KIT With OV2640 Module"
 config CAMERA_MODEL_M5STACK_PSRAM
     bool "M5Stack Camera With PSRAM"
+config CAMERA_MODEL_ESP32_CAM
```

Comment:
please rename this to `CAMERA_MODEL_AI_THINKER` :) `CAMERA_MODEL_ESP32_CAM` is a bit too generic :)

Reply:
done